File: Site/controller/CUtilisateur.php
<?php 
    include("modele/Utilisateur.php");

    function connexionUtilisateur($pseudo){
        return Utilisateur::connexionUtilisateur($pseudo);
    }
    function inscriptionUtilisateur($pseudo){
        return Utilisateur::inscriptionUtilisateur($pseudo);
    }
?>

File: Serveur/controller/CUtilisateur.py
from flask import jsonify
from modele.Utilisateur import Utilisateur

def connexion(request):
    try:
        pseudo = request.form.get('pseudo')
        return Utilisateur.connexionUtilisateur(pseudo)
    except:
        return jsonify(error=True)


def inscription(request):
    try:
        pseudo = request.form.get('pseudo')

        return Utilisateur.inscriptionUtilisateur(pseudo)

    except:
        return jsonify(error = True)